package.json

```diff
@@ -93,8 +93,6 @@
     "extensionizer": "^1.0.0",
     "fast-json-patch": "^2.0.4",
     "fast-levenshtein": "^2.0.6",
-    "gulp": "github:gulpjs/gulp#4.0",
```

Comment:
We don't need gulp anymore? Isn't this useful/important, esp. since it's a specific version?

Reply:
sorry should have clarified!
 gulp was present in both dev-dep and regular deps. this removes it from regular deps.